src/main/resources/db/migration/V0.0.2_001__create_questionaire.sql

```diff
@@ -54,4 +54,4 @@ add constraint FKjqvqo75p8dpn6oweuepac4ubf foreign key (questionnaire_id) refere
 alter table if exists question
 add constraint FK7jaqbm9p4prg7n91dd1uabrvj foreign key (category_id) references category;
 alter table if exists single_option
-add constraint FKi8whcw3jwcfm8sklck50a6g0e foreign key (question_id) references question;
\ No newline at end of file
+add constraint FKi8whcw3jwcfm8sklck50a6g0e foreign key (question_id) references question;
```

Comment:
we don't need to change this file just for a new line at the end

Reply:
I simply added EOF character